src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx.cs

```diff
@@ -14,223 +14,223 @@ namespace System.Runtime.Intrinsics.X86
-        /// __m256d _mm256_cmp_pd (__m256d a, __m256d b, const int imm8)
+        /// __m256d _mm256_cmp_pd (__m256d a, __m256d b, const int imm8); VCMPPD ymm, ymm, ymm/m256, imm8
         /// </summary>
         public static Vector256<double> Compare(Vector256<double> left, Vector256<double> right, FloatComparisonMode mode) => Compare(left, right, mode);
 
         /// <summary>
-        /// __m128d _mm_cmp_sd (__m128d a, __m128d b, const int imm8)
+        /// __m128d _mm_cmp_sd (__m128d a, __m128d b, const int imm8); VCMPSS xmm, xmm, xmm/m32, imm8
         /// </summary>
         public static Vector128<double> CompareScalar(Vector128<double> left, Vector128<double> right, FloatComparisonMode mode) => CompareScalar(left, right, mode);
         /// <summary>
-        /// __m128 _mm_cmp_ss (__m128 a, __m128 b, const int imm8)
+        /// __m128 _mm_cmp_ss (__m128 a, __m128 b, const int imm8); VCMPSD xmm, xmm, xmm/m64, imm8
         /// </summary>
         public static Vector128<float> CompareScalar(Vector128<float> left, Vector128<float> right, FloatComparisonMode mode) => CompareScalar(left, right, mode);
 
         /// <summary>
-        /// float _mm256_cvtss_f32 (__m256 a)
+        /// float _mm256_cvtss_f32 (__m256 a); HELPER: VMOVSS
         /// </summary>
         public static float ConvertToSingle(Vector256<float> value) => ConvertToSingle(value);
 
         /// <summary>
-        /// __m128i _mm256_cvtpd_epi32 (__m256d a)
+        /// __m128i _mm256_cvtpd_epi32 (__m256d a); VCVTPD2DQ xmm, ymm/m256
         /// </summary>
         public static Vector128<int> ConvertToVector128Int32(Vector256<double> value) => ConvertToVector128Int32(value);
         /// <summary>
-        /// __m128 _mm256_cvtpd_ps (__m256d a)
+        /// __m128 _mm256_cvtpd_ps (__m256d a); VCVTPD2PS xmm, ymm/m256
         /// </summary>
         public static Vector128<float> ConvertToVector128Single(Vector256<double> value) => ConvertToVector128Single(value);
         /// <summary>
-        /// __m256i _mm256_cvtps_epi32 (__m256 a)
+        /// __m256i _mm256_cvtps_epi32 (__m256 a); VCVTPS2DQ ymm, ymm/m256
         /// </summary>
         public static Vector256<int> ConvertToVector256Int32(Vector256<float> value) => ConvertToVector256Int32(value);
         /// <summary>
-        /// __m256 _mm256_cvtepi32_ps (__m256i a)
+        /// __m256 _mm256_cvtepi32_ps (__m256i a); VCVTDQ2PS ymm, ymm/m256
         /// </summary>
         public static Vector256<float> ConvertToVector256Single(Vector256<int> value) => ConvertToVector256Single(value);
         /// <summary>
-        /// __m256d _mm256_cvtps_pd (__m128 a)
+        /// __m256d _mm256_cvtps_pd (__m128 a); VCVTPS2PD ymm, xmm/m128
         /// </summary>
-        public static Vector256<double> ConvertToVector256Double(Vector256<float> value) => ConvertToVector256Double(value);
+        public static Vector256<double> ConvertToVector256Double(Vector128<float> value) => ConvertToVector256Double(value);
```

Comment:
The parameter was incorrect, changed from `Vector256` to `Vector128`

Reply:
Thank you!!